app/controllers/bands_controller.rb

```diff
@@ -1,10 +1,46 @@
 class BandsController < InheritedResources::Base
 
+  before_action :find_band, except: [:index, :new, :create]
```

Comment:
This controller looks good but, what about if instead of use `except` we use `only`, I that way I think is more readable. Opinions?